### PR TITLE
Adds latest support from multishot.

### PR DESCRIPTION
**Pull request by dawnerd**
Fixes for #2 #3 #4

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -5,8 +5,44 @@ module.exports = function(grunt) {
       default: {
         temp: '/tmp/multishot/',
         output: '/tmp/multishot/',
-        url: ['http://google.com', 'http://yahoo.com', 'http://facebook.com', 'http://digg.com', 'http://amazon.com']
-        // file: './node_modules/multishot/example/urls.txt'
+        url: ['http://google.com', 'http://yahoo.com']
+      },
+      prefix: {
+        temp: '/tmp/multishot/prefix/',
```
Comment (jgallen23) [inline Gruntfile.js:11]: temp is optional, right?

Comment (dawnerd) [inline Gruntfile.js:11]: Yep

**Comment by jgallen23** [comment]
feel free to publish to npm

**Comment by jgallen23** [comment]
but make sure you update the History.md file

**Pull request merged**